README.md

```diff
@@ -0,0 +1,21 @@
+## Insights service
+
+* This repository builds the opni-insights image which sets up an HTTP server that will return the breakdown of log messages by pod, namespace and workload.
```

Comment:
can you add some sample documentation on how one would call any of the api endpoints?

e.g.
```
curl --location --request GET 'localhost:8082/workload?start_ts=1631794584000&end_ts=1631855784000' \
--header 'Content-Type: application/json'
```